### PR TITLE
[QQC-1484] Support move to task action

### PR DESCRIPTION
**Pull request by richardsun0713**
* Added new entity TaskQueue
* Added new methods in Project:
  * Obtain a list of TaskQueues owned by the Project
  * Move a list of data rows to a specified TaskQueue

### CODE REVIEW

File: labelbox/schema/project.py [hunk 1]
```diff
@@ -1127,6 +1108,99 @@ def batches(self) -> PaginatedCollection:
...
+                "dataRowIds": data_row_ids
+            },
+            timeout=180.0,
+            experimental=True)["project"][method]["taskId"]
+
+        status = self._wait_for_task(task_id)
+        if status != "COMPLETE":
+            raise LabelboxError(f"Data rows were not moved successfully")
+
+    def _wait_for_task(self, task_id: str):
```
Comment (kkim-labelbox) [inline labelbox/schema/project.py:1179]: Have you considered using our existing Task class for this? `_wait_for_task` looks similar to `Task.refresh()`

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -359,6 +359,66 @@ def create_label():
...
+                                        batch_project, dataset, datarow,
+                                        wait_for_label_processing):
+    """Project with a batch having one datarow
+    Project contains an ontology with 1 bbox tool
+    Additionally includes a create_label method for any needed extra labels
+    One label is already created and yielded when using fixture
+    """
+    data_rows = [dr.uid for dr in list(dataset.data_rows())]
+    batch_project.create_batch("test-batch", data_rows)
+    editor = list(
```
Comment (kkim-labelbox) [inline tests/integration/conftest.py:373]: Looks like a lot of the code after this is shared between this and `configured_project_with_label`. Could you create a helper function for the ontology setup + label creation part?

File: labelbox/schema/project.py [hunk 3]
```diff
@@ -1127,6 +1108,99 @@ def batches(self) -> PaginatedCollection:
...
+            query_str, {"projectId": self.uid},
+            timeout=180.0,
+            experimental=True)["project"]["taskQueues"]
+
+        return [
+            Entity.TaskQueue(self.client, field_values)
+            for field_values in task_queue_values
+        ]
+
+    def move_data_rows_to_task(self, data_row_ids: List[str],
```
Comment (kkim-labelbox) [inline labelbox/schema/project.py:1136]: nit: `move_data_rows_to_task_queue` sounds more accurate

File: labelbox/schema/project.py [hunk 4]
```diff
@@ -1127,6 +1108,99 @@ def batches(self) -> PaginatedCollection:
...
+                "projectId": self.uid,
+                "queueId": task_queue_id,
+                "dataRowIds": data_row_ids
+            },
+            timeout=180.0,
+            experimental=True)["project"][method]["taskId"]
+
+        status = self._wait_for_task(task_id)
+        if status != "COMPLETE":
+            raise LabelboxError(f"Data rows were not moved successfully")
```
Comment (kkim-labelbox) [inline labelbox/schema/project.py:1177]: Is there a way we can show the error message from the task to users?

File: labelbox/schema/task_queue.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class TaskQueue(DbObject):
```
Comment (kkim-labelbox) [inline labelbox/schema/task_queue.py:5]: Please add this class as an entry in `docs/source/index.rst` to be indexed for readthedocs.
More details https://labelbox.atlassian.net/wiki/spaces/AIENG/pages/1721139216/Release+a+new+Python+SDK+Version#Documentation-Updates

File: labelbox/schema/project.py (hunk 1 above)
Comment (richardsun0713) [reply labelbox/schema/project.py:1179]: The Task class is specialized to handle data rows as the result type, but I can make it work without breaking existing usages

File: labelbox/schema/project.py (hunk 3 above)
Comment (richardsun0713) [reply labelbox/schema/project.py:1136]: I'll update it to `move_data_rows_to_task_queue`, but keep the action as `move` for consistency with the UI equivalent

File: tests/integration/conftest.py [hunk 6]
```diff
@@ -311,47 +311,9 @@ def configured_project_with_label(client, rand_gen, image_url, project, dataset,
...
-        upload_task.wait_until_done(sleep_time_seconds=5)
-        assert upload_task.state == AnnotationImportState.FINISHED, "Label Import did not finish"
-        assert len(
-            upload_task.errors
-        ) == 0, f"Label Import {upload_task.name} failed with errors {upload_task.errors}"
 
-    project.create_label = create_label
-    project.create_label()
-    label = wait_for_label_processing(project)[0]
+    ontology = _setup_ontology(project)
```
Comment (kkim-labelbox) [inline tests/integration/conftest.py:315]: So clean!

File: labelbox/schema/project.py [hunk 7]
```diff
@@ -1172,34 +1173,16 @@ def move_data_rows_to_task(self, data_row_ids: List[str],
             timeout=180.0,
             experimental=True)["project"][method]["taskId"]
 
-        status = self._wait_for_task(task_id)
-        if status != "COMPLETE":
-            raise LabelboxError(f"Data rows were not moved successfully")
+        task = self._wait_for_task(task_id)
+        if task.status != "COMPLETE":
+            raise LabelboxError(f"Data rows were not moved successfully: " +
```
Comment (kkim-labelbox) [inline labelbox/schema/project.py:1178]: What type of Task is this job? Currently, Task.errors only returns value for a `JSON Import` task

**Review by kkim-labelbox — APPROVED** [review]
Code looks good to me. Please resolve the merge conflict, and make sure the SDK tests run

**Pull request merged**